### PR TITLE
events.cancel

### PR DESCRIPTION
**Pull request by psrpinto**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 PWD = $(shell pwd)
 
-all: lint build
+all: lint build preview
```
Comment (jenssegers) [inline Makefile]: Can we revert this? I never use preview, not really needed as a default make target

Comment (jenssegers) [inline Makefile]: Can we revert this please?

**Review by jenssegers — DISMISSED** [review]

File: Makefile (hunk 1 above)
Comment (psrpinto) [reply Makefile]: I'll revert but I think it makes sense to also generate the preview. For me at least, I test it locally so I always have to run make twice.

 :ok: by @regularjack on 24/04/2018 11:13:51

**Comment by psrpinto** [comment]
@jenssegers Can you re-review? Addressed your remark

**Review by mrtus — APPROVED** [review]

**Review by Jdww — APPROVED** [review]

**Review by jenssegers — APPROVED** [review]

**Pull request merged**